test/e2e/storage/csi_mock/csi_workload.go

```diff
@@ -45,13 +45,6 @@ var _ = utils.SIGDescribe("CSI Mock workload info", func() {
 			expectPodInfo          bool
 			expectEphemeral        bool
 		}{
-			{
```

Comment:
I remove this case because `csidriver.spec.podInfoOnMount` cannot be nil. 

```
  podInfoOnMount	<boolean>
    podInfoOnMount indicates this CSI volume driver requires additional pod
    information (like podName, podUID, etc.) during mount operations, if set to
    true. If set to false, pod information will not be passed on mount. Default
    is false.
```

Reply:
Can you then please change `podInfoOnMount *bool` to just `bool` on line 43? It should look slightly better then.

Reply:
/cc @jsafrane 

updated. please review again.